src/Conveyer.ts

```diff
@@ -297,8 +298,22 @@ class Conveyer extends Component<ConveyerEvents> {
     const scrollAreaElement = this.scrollAreaElement;
     const horizontal = this.options.horizontal;
 
-    this.size = horizontal ? scrollAreaElement.clientWidth : scrollAreaElement.clientHeight;
-    this.scrollSize = horizontal ? scrollAreaElement.scrollWidth : scrollAreaElement.scrollHeight;
+    const size = horizontal ? scrollAreaElement.clientWidth : scrollAreaElement.clientHeight;
+    let scrollSize = horizontal ? scrollAreaElement.scrollWidth : scrollAreaElement.scrollHeight;
+
+    // check decimal point
+    if (IS_IE && scrollSize === size + 1) {
+      const offsetSize = horizontal ? scrollAreaElement.offsetWidth : scrollAreaElement.offsetHeight;
+      const rect = scrollAreaElement.getBoundingClientRect();
+      const boundingSize = horizontal ? rect.width : rect.height;
+
+      // 0 < 0.5 && 0.5 < 1
+      if (offsetSize < boundingSize && boundingSize < offsetSize + 1) {
+        scrollSize = size;
+      }
+    }
+    this.size = size;
+    this.scrollSize = scrollSize;
```

Comment:
I'm just being curious, why do you style protected values like public values?
I mean, those values are internal, not open to the users, so they should be treated like private values.
In short, how about adding underscore prefix(`_`) to those protected values/methods?

Reply:
protected is not private even if used internally. protected can be converted to public. That's why I didn't use underscore. Discussion is needed.

Reply:
Looks so.
I'm treating `protected` properties more like "private but can be inherited" (I think this is quite typical in OOP), so I don't get it about "protected can be converted to public".
Let's talk about it tomorrow :)